app/dashboard/management/commands/cleanup_db_space.py

```diff
@@ -56,8 +56,3 @@ def handle(self, *args, **options):
                 created_on__hour=1,
             ).delete()
         print(f'Stat: {result}')
-
-        result = LeaderboardRank.objects.filter(
```

Comment:
hmmm m we should stop doing this now.. gonna fix that here d84147cfa3d4245589949d06a6799ba3c99b83fd